rllib/agents/trainer.py

```diff
@@ -106,6 +106,9 @@
     "clip_rewards": None,
     # Whether to np.clip() actions to the action space low/high range spec.
     "clip_actions": True,
+    # Whether to argmax actions (to translate from a Trainer Box action space
+    # to an Env Discrete action space).
+    "argmax_actions": False,
```

Comment:
I think this is now unused.

Reply:
+1

Reply:
Good catch. This would have actually broken the `Trainer.compute_action` calls :/
I will add this to the tests.